ci-operator/config/ComplianceAsCode/content/ComplianceAsCode-content-master__4.15.yaml

```diff
@@ -810,6 +810,106 @@ tests:
         requests:
           cpu: 100m
     workflow: ipi-aws
+- as: e2e-aws-rosa-ocp4-pci-dss-node-weekly
+  cron: 0 22 * * 6
+  steps:
+    cluster_profile: quay-aws
+    env:
+      ADDITIONAL_SECURITY_GROUP: "true"
+      CHANNEL_GROUP: nightly
+      OCM_LOGIN_ENV: integration
+      OPENSHIFT_VERSION: "4.15"
+    test:
+    - as: test-ocp4-pci-dss-node
+      cli: latest
+      commands: |
+        export PROFILE=pci-dss-node
+        export PRODUCT=ocp4
+        export CONTENT_IMAGE=ghcr.io/complianceascode/k8scontent:latest
+        export component=ocp4-content-ds
+        git clone https://github.com/ComplianceAsCode/ocp4e2e.git ocp4e2e
+        pushd ocp4e2e
+        go test -v -timeout 120m github.com/ComplianceAsCode/ocp4e2e -profile="$PROFILE" -product="$PRODUCT" -content-image="$CONTENT_IMAGE"
+      from: src
+      resources:
+        requests:
+          cpu: 100m
+    workflow: rosa-aws-sts-hcp
+- as: e2e-aws-rosa-ocp4-cis-weekly
+  cron: 0 22 * * 6
+  steps:
+    cluster_profile: quay-aws
+    env:
+      ADDITIONAL_SECURITY_GROUP: "true"
+      CHANNEL_GROUP: nightly
+      OCM_LOGIN_ENV: integration
+      OPENSHIFT_VERSION: "4.15"
```

Comment:
In my testing with https://github.com/openshift/release/pull/50090 I needed to update this to `4.15.8` and change the CHANNEL_GROUP to `stable`.

The nightly channel group seemed to have more specific version strings (4.15.8 wasn't an option).

Reply:
Correct. I will update to use stable build instead.